src/styles.scss

```diff
@@ -21,4 +22,51 @@ html, body {
     justify-content: center;
     flex-direction: column;
     height: 100%;
+}
+
+/* FONTS */
+
+.frank{
+    font-family: 'Frank Ruhl Libre', serif;
+    font-style: normal;
+}
+
+.frank-picture-header{
+    @extend .frank;
+    font-weight: bold;
+    font-size: 115px;
+    line-height: 124px;
+}
+
+.frank-small-header{
+    @extend .frank;
+    font-weight: bold;
+    font-size: 64px;
+    line-height: 83px;
+}
```

Comment:
thoughts on specifying margins for our styling?

Reply:
The default margin seems to be good in most cases except for headings so we can maybe do something about that. Do you have any ideas for default values?